core-services/prow/02_config/_labels.yaml

```diff
@@ -653,6 +653,14 @@ repos:
         name: docs-approved
         target: both
         addedBy: label
+  openshift/compliance-operator:
```

Comment:
The labels need to be under `labels`, and the items are missing target (prs? issues? both?) and who adds the label

Reply:
Fixing, thanks!

Reply:
Looks like you added the missing fields to only one label, not both. Probalby just an omission?

Reply:
just me not being able to copy/paste appropriately...